src/main/java/Deadline.java

```diff
@@ -0,0 +1,24 @@
+public class Deadline extends Task{
+    String by;
+
+    public Deadline(String task, String by){
+        super(task);
+        this.by = by;
+    }
+
+    public String toString(){
+        if(this.isDone()){
+            return ("[D][X] " + this.getTask() + " (by: " + this.getBy() +")\n");
+        }else{
+            return ("[D][ ] " + this.getTask() + " (by: " + this.getBy() +")\n");
```

Comment:
You might want to consider using the status of the object(done/ not done yet) as an indicator of the cross.